src/Api/HL/Router.php

```diff
@@ -579,37 +579,41 @@ public function handleRequest(Request $request): Response
      */
     private function handleAuth(Request $request): void
     {
-        if ($request->hasHeader('Authorization')) {
-            $auth_header = $request->getHeaderLine('Authorization');
-            // if basic auth
-            if (preg_match('/^Basic\s+(.*)$/i', $auth_header, $matches)) {
-                $this->resumeSession('');
-            } else {
-                $this->startTemporarySession($request);
-                if ($request->hasHeader('GLPI-Profile')) {
-                    $requested_profile = $request->getHeaderLine('GLPI-Profile');
-                    if (is_numeric($requested_profile)) {
-                        Session::changeProfile((int) $requested_profile);
+        // Try authenticating from a cookie for cases when the API is accessed from a browser
```

Comment:
In the past, I already had to deal with the support related to the fact that a user had a different behavior between the requests launched from his browser via Postman and the requests launched from a PHP script. These differences were because Postman returned the cookie in subsequent requests, while the PHP script did not.

I do not think we have this problem with GLPI, but if it is possible not to send these cookies, it would be better. Maybe the documentation endpoints could have a middleware that transforms session cookie into a `Glpi-Session-Token` header, so it would be the only endpoints that would be able to resume session using cookies.

Also, it would be better to find a solution to send cookies only for documentation endpoints responses.